1-Initial-Setup/1.6-Additional-Process-Hardening.bats

```diff
@@ -26,8 +26,8 @@
         subfile_is_hard_core=$?
     fi
 
-    run bash -c "grep 'hard core' /etc/security/limits.conf"
-    [ "$output" == "* hard core 0" ]
+    run bash -c "grep 'hard core' /etc/security/limits.conf /etc/security/limits.d/*"
+    [[ "$output" == *"* hard core 0" ]]
```

Comment:
Maybe double bracket notation is discouraged? (https://github.com/sstephenson/bats/issues/49)
Can you check if we are affected from this?

Reply:
As mentioned in https://github.com/sstephenson/bats/issues/49#issuecomment-48608117 this is fixed with the recent version of bash we are using, so this won't affect us.